src/System.IO.FileSystem.DriveInfo/src/System.IO.FileSystem.DriveInfo.csproj

```diff
@@ -73,19 +73,19 @@
     <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
       <Link>Common\Interop\Unix\Interop.Libraries.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\CoreLib\Interop\Unix\Interop.Errors.cs">
-      <Link>Common\CoreLib\Interop\Unix\Interop.Errors.cs</Link>
+    <Compile Include="$(CoreLibDir)Interop\Unix\Interop.Errors.cs">
+      <Link>Interop\Unix\Interop.Errors.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\CoreLib\Interop\Unix\Interop.IOErrors.cs">
-      <Link>Common\CoreLib\Interop\Unix\Interop.IOErrors.cs</Link>
+    <Compile Include="$(CoreLibDir)Interop\Unix\Interop.IOErrors.cs">
+      <Link>Interop\Unix\Interop.IOErrors.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\System\IO\PathInternal.Unix.cs">
       <Link>Common\System\IO\PathInternal.Unix.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\CoreLib\Interop\Unix\System.Native\Interop.PathConf.cs">
+    <Compile Include="$(CoreLibDir)Interop\Unix\System.Native\Interop.PathConf.cs">
       <Link>Common\Interop\Unix\Interop.PathConf.cs</Link>
```

Comment:
It seems like this Link didn't contain  `CoreLib` so it still contains the `Common` prefix.